src/core/components/auth/authorization-popup-perAPI.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default class AuthorizationPopupPerAPI extends React.Component {
+  close =() => {
```

Comment:
Do we need this? I don't see anything in the rendered view calling this function.

Reply:
No,
I will remove it

Reply:
Resolved.